amundsen_application/static/js/components/ColumnList/index.tsx

```diff
@@ -153,17 +197,26 @@ const ColumnList: React.FC<ColumnListProps> = ({
         name: item.name,
         database,
       },
-      usage: hasItemStats ? item.stats[0].stat_val : '',
+      sort_order: item.sort_order,
+      usage: hasItemStats ? +item.stats[0].stat_val : null,
       stats: hasItemStats ? item.stats[0] : null,
       action: item.name,
+      name: item.name,
       isEditable: item.is_editable,
       editText,
       editUrl,
       index,
     };
   });
   const statsCount = formattedData.filter((item) => !!item.stats).length;
-  const hasStats = statsCount >= SHOW_STATS_THRESHOLD;
+  const hasStats =
```

Comment:
nit: Is this more accurately `hasUsage`?

Reply:
Yep, will change